### PR TITLE
Coverage overload

### PR DESCRIPTION
**Pull request by fredrikekre**
Adds more constructor tests, and some other stuff too.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -5,56 +5,139 @@ using ContMechTensors
...
+            @test I[i,j] == T(1)
+            @test I_sym[i,j] == T(1)
+        else
+            @test I[i,j] == T(0)
+            @test I_sym[i,j] == T(0)
+        end
+        for k in 1:dim, l in 1:dim
+            if i == k && j == l
+                @test II[i,j,k,l] == T(1)
+                # @test II_sym[i,j,k,l] == T(1)
```
Comment (fredrikekre) [inline test/runtests.jl:97]: whats up with these not working? :(

File: test/runtests.jl [hunk 2]
```diff
@@ -5,56 +5,139 @@ using ContMechTensors
...
+
+        # Binary tensor tensor: +, -
+        @test (@inferred t + t) == 2 * t == 2 * Array(t)
+        @test isa(t + t, $TensorType{$order, $dim})
+
+        @test (@inferred t - t) == zero(t) == 0 * Array(t)
+        @test isa(t - t, $TensorType{$order, $dim})
+
+        # Binary tensor number: +, -, *, /
+        # @test 2 + t == t + 2 == 2 + Array(t)
```
Comment (fredrikekre) [inline test/runtests.jl:121]: Will add this in another PR

File: test/test_ops.jl [hunk 3]
```diff
@@ -222,17 +224,10 @@ end # of testset
     @test AA_sym[i,j,k,l] ≈ majortranspose(AA_sym)[k,l,i,j]
     @test typeof(majortranspose(AA_sym)) <: Tensor{4,dim}
 
-    @test _permutedims(AA,(1,2,3,4)) ≈ AA
```
Comment (fredrikekre) [inline test/test_ops.jl:225]: no need to test this now that its only a helper function

File: test/runtests.jl [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Build the real docs first.
-include("../docs/make.jl")
+# include("../docs/make.jl")
```
Comment (KristofferC) [inline test/runtests.jl]: Can haz coverage?

Comment (fredrikekre) [reply test/runtests.jl]: plz

Comment (fredrikekre) [reply test/runtests.jl]: Its gonna time out anyway :p Maybe the problem is not the doc building?

Comment (KristofferC) [reply test/runtests.jl]: I think it is two problems. Doc building and deprecations in stuff needed for coverage.

Comment (KristofferC) [reply test/runtests.jl]: You can let the 0.5 worker do coverage instead.

Comment (fredrikekre) [reply test/runtests.jl]: Lets try.

**Comment by fredrikekre** [comment]
Sucess!

**Comment by KristofferC** [comment]
lol

**Comment by fredrikekre** [comment]
:(

**Comment by KristofferC** [comment]
Coverage kan vara bättre på master, det var några PRs om det.

**Comment by fredrikekre** [comment]
Should also merge #73 to increase it, will do that now

**Comment by fredrikekre** [comment]
Seems like the Travis problems are fixed!

**Comment by KristofferC** [comment]
Nice. New version of Documenter tagged.

**Comment by fredrikekre** [comment]
I thought we checkout master, but maybe you changed that when we include the doc build in the test suite?

**Comment by KristofferC** [comment]
Yeah I changed it.

**Comment by fredrikekre** [comment]
> lol

?

**Comment by KristofferC** [comment]
:tada:

**Pull request merged**